docs/modules/operation/pages/deep-dive/events/event-tokens.adoc

```diff
@@ -1,121 +1,174 @@
 
-A node may have additional asset records stored for it.
-You can access these records using the `asset` replacement token, which takes the form:
+== Tokens
 
-%asset[<token>]%::
-    The value of the asset field named `token`, or "Unknown" if it does not exist.
+You can use the following replacement tokens in {page-component-title}:
 
-== Hardware tokens
+[options="autowidth"]
+|===
+| Parameter | Description
+
+| %eventid%
+| Event's numeric database ID.
+
+| %uei%
+| Event's unique event identifier.
+
+| %source%
+| Source of the event (which {page-component-title} daemon created it).
+
+| %descr%
+| Event description.
+
+| %logmsg%
+| Event's log message.
+
+| %time%
+| The time at which the event occurred.
+
+| %shorttime%
+| The time at which the event occurred, formatted as `DateFormat.SHORT`.
+This provides a completely numeric date and time.
+
+| %dpname%
+| Formerly used for the ID of the Minion that the event was received on.
+This placeholder is deprecated and resolves to an empty string.
+
+| %nodeid%
+| Numeric node ID of the device that caused the event, if any.
+
+| %nodelabel%
+| Label for the node identified by `%nodeid%`, if available.
+
+| %nodelocation%
+| Location for the node identified by `%nodeid%`, if available.
+
+| %host%
+| The host on which the event was generated.
+
+| %interface%
+| IP interface associated with the event, if any.
+
+| %foreignsource%
+| Requisition name for the node identified by `%nodeid%`, if available.
+
+| %ifindex%
+| The interface's SNMP `ifIndex`.
+
+| %interfaceresolve%
+| Performs a reverse lookup on the `%interface%` parameter and returns its name, if available.
+
+| %service%
+| Service associated with the event, if any.
+
+| %severity%
+| Event's severity level.
+
+| %snmphost%
+| The host of the SNMP agent that generated the event.
+
+| %id%
+| Event's SNMP enterprise OID.
+
+| %trapoid%
+| Event's full trap OID.
+
+| %idtext%
+| Event's decoded (human-readable) SNMP enterprise OID.
+
+| %ifalias%
+| The interface's SNMP `ifAlias`.
+
+| %generic%
+| Event's generic trap-type number.
 
-A node may have additional hardware details stored for it.
-You can access these details using the `hardware` replacement token, which takes the form:
+| %specific%
+| Event's specific trap-type number.
 
-%hardware[<token>]%::
-    The value of the `token` field from the node's hardware inventory.
+| %community%
+| Trap's community string.
+
+| %version%
+| Trap's SNMP version.
+
+| %snmp%
+| The SNMP information associated with the event.
+
+| %operinstruct%
+| Event's operator instructions.
+
+| %mouseovertext%
+| Event's mouse-over text.
+
+| %tticketid%
+| Event's trouble ticket ID, if available.
+
+| %primaryinterface%
+| The primary interface IP address for the node identified by `%nodeid%`, if available.
+|===
+
+CAUTION: If you use multiple Minions in one location, it can break the alarm lifecycle for some {page-component-title} features.
```

Comment:
Curious about the caution, specifically line 119. If `%dpname%` is deprecated, do we want to suggest using it? It resolves to an empty string as well, so replacing it with an empty string seems redundant?

If we keep this, suggest the following:

```suggestion
CAUTION: Using multiple Minions in one location can break the alarm lifecycle for some {page-component-title} features.
```
With the other syntax, it wasn't clear what was breaking the alarm cycle.